src/components/widgets/TextureWidget.js

```diff
@@ -178,7 +178,7 @@ export default class TextureWidget extends React.Component {
       <span className="texture">
         <span className={this.state.valueType}></span>
         <canvas ref="canvas" width="32" height="16" title={this.props.mapName} onClick={this.openDialog}></canvas>
-        <input className="map_value" type="text" value={this.state.value} onChange={this.onChange}/>
+        <input className="map_value string" type="text" value={this.state.value} onChange={this.onChange}/>
```

Comment:
the underscore is inconsistent with other class names in this project. what shall I change it to?